CSharp/DevVmPowershell/Helpers/Implementations/InstanceSettingsHelper.cs

```diff
@@ -48,33 +74,72 @@ public bool UpdateInstanceSettingValue(string name, string section, string newVa
 				{
 					newValue = string.Empty;
 				}
-				using (IInstanceSettingManager instanceSettingManager = ServiceFactory.CreateProxy<IInstanceSettingManager>())
+
+				HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+				string queryUrl = $"Relativity.REST/api/Relativity.Objects/workspace/{Constants.EDDS_WORKSPACE_ARTIFACT_ID}/object/query";
+				ObjectManagerQueryRequestModel objectManagerQueryRequestModel = new ObjectManagerQueryRequestModel
 				{
-					Query query = new Query();
-					query.Condition = $"'Section' == '{section}' AND 'Name' == '{name}'";
-					InstanceSettingQueryResultSet instanceSettingQueryResultSet = instanceSettingManager.QueryAsync(query).Result;
-					if (instanceSettingQueryResultSet.Success)
+					request = new Request
 					{
-						Relativity.Services.Result<InstanceSetting> result = instanceSettingQueryResultSet.Results.First();
-						if (result.Artifact != null)
+						objectType = new Helpers.RequestModels.ObjectType
+						{
+							Name = Constants.InstanceSetting.INSTANCE_SETTING_OBJECT_TYPE
+						},
+						fields = new object[]
 						{
-							Console.WriteLine("Successfully found the existing Instance Setting");
-							InstanceSetting instanceSetting = result.Artifact;
-							if (instanceSetting.Name == "ESIndexCreationSettings")
+							new
 							{
-								string previousValue = instanceSetting.Value;
-								newValue = previousValue.Replace("\"number_of_shards\": 12", "\"number_of_shards\": 2");
-								newValue = newValue.Replace("\"number_of_replicas\": 2", "\"number_of_replicas\": 0");
+								Name = "Name"
+							},
+							new
+							{
+								Name = "Value"
 							}
-							instanceSetting.Value = newValue;
-							instanceSettingManager.UpdateSingleAsync(instanceSetting).Wait();
-							Console.WriteLine("Successfully updated the Instance Setting");
-							return true;
+						},
+						condition = $"'Section' == '{section}' AND 'Name' == '{name}'"
+					},
+					start = 1,
+					length = 10
+				};
+
+				string queryRequest = JsonConvert.SerializeObject(objectManagerQueryRequestModel);
+				HttpResponseMessage queryResponse = RestHelper.MakePost(httpClient, queryUrl, queryRequest);
+				if (!queryResponse.IsSuccessStatusCode)
+				{
+					throw new Exception("Failed to Query for Agent Artifact Ids");
+				}
+				string result = queryResponse.Content.ReadAsStringAsync().Result;
+				JObject jObject = JObject.Parse(result);
```

Comment:
Thoughts on deserializing into a model vs the more manual current process?  Admittedly, the pasting the current JSON as a model might seem like it's cleaning it up but it could just end up with more things to maintain.

Reply:
@TommyBigNose What current manual process?

Reply:
@aarongilbert123 sorry poor choice of words, the following code where it's a lot of stuff like this: jObject["Objects"][0]["FieldValues"][0]["Value"].Value<string>();

Reply:
@TommyBigNose Yeah I feel like that just may overcomplicate things. Might be hard to maintain if those responses change in the future

Reply:
Was my concern as well which is why I made it a discussion than a request.  Cool, thanks